content/zh-cn/docs/tasks/tools/install-kubectl-linux.md

```diff
@@ -56,10 +56,10 @@ The following methods exist for installing kubectl on Linux:
 
    {{< tabs name="download_binary_linux" >}}
    {{< tab name="x86-64" codelang="bash" >}}
-   curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl.sha256"
+curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
```

Comment:
The indentations are good, why remove them?

Reply:
LGTM

IMO it looks good without indentation. The Indentation is annoying when you paste the command into the terminal.

Here are the links for a quick comparison:
https://deploy-preview-43430--kubernetes-io-main-staging.netlify.app/docs/tasks/tools/install-kubectl-linux/#install-kubectl-binary-with-curl-on-linux

https://deploy-preview-43430--kubernetes-io-main-staging.netlify.app/docs/tasks/tools/install-kubectl-linux/#install-kubectl-binary-with-curl-on-linux

Reply:
> The indentations are good, why remove them?



> LGTM
> 
> IMO it looks good without indentation. The Indentation is annoying when you paste the command into the terminal.
> 
> Here are the links for a quick comparison: https://deploy-preview-43430--kubernetes-io-main-staging.netlify.app/docs/tasks/tools/install-kubectl-linux/#install-kubectl-binary-with-curl-on-linux
> 
> https://deploy-preview-43430--kubernetes-io-main-staging.netlify.app/docs/tasks/tools/install-kubectl-linux/#install-kubectl-binary-with-curl-on-linux

Yes, that's what I mean.

Reply:
Well ... looks like we are trapped by the "whole tab snippet" issue again.
Looks like the `tab` doesn't handle indentation well, you have to unindent whatever inside the block. That is annoying.